data/net/minecraft/data/worldgen/placement/PlacementUtils.mapping

```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/data/worldgen/placement/PlacementUtils
+	METHOD filtered (Lnet/minecraft/world/level/levelgen/feature/Feature;Lnet/minecraft/world/level/levelgen/feature/configurations/FeatureConfiguration;Lnet/minecraft/world/level/levelgen/blockpredicates/BlockPredicate;)Lnet/minecraft/core/Holder;
+		ARG 0 feature
+		ARG 1 featureConfiguration
+	METHOD filteredByBlockSurvival (Lnet/minecraft/world/level/block/Block;)Lnet/minecraft/world/level/levelgen/placement/BlockPredicateFilter;
+		ARG 0 block
+	METHOD inlinePlaced (Lnet/minecraft/core/Holder;[Lnet/minecraft/world/level/levelgen/placement/PlacementModifier;)Lnet/minecraft/core/Holder;
+		ARG 0 feature
+		ARG 1 modifiers
```

Comment:
```suggestion
		ARG 1 placements
```

In the same vein as above, this bakes down to a `PlacedFeature`, of which the field is `placement`. (Notably, not plural, despite being a list of `PlacementModifier`s.

I would suggest `placements` here, because the plural makes sense for a varadic argument, although `placement` would also work.